src/main/java/net/minecraftforge/fml/client/FMLClientHandler.java

```diff
@@ -433,6 +434,14 @@ public void onInitializationComplete()
         else
         {
             logMissingTextureErrors();
+            if (!badTextureDomains.isEmpty() || !badResources.isEmpty())
+            {
+                if ((Boolean) Launch.blackboard.get("fml.deobfuscatedEnvironment")) {
```

Comment:
NPEs?

Reply:
All occurences of this check are written like this - but might as well change it